applications/wg-easy/Taskfile.yaml

```diff
@@ -56,6 +56,15 @@ tasks:
         vars:
           TIMEOUT: "{{.TIMEOUT}}"
 
+  list-cluster:
```

Comment:
How is this different from `replicated cluster ls`?
Or did you just like having it in Taskfile for consistency?

Reply:
replicated cluster ls will list all the clusters, not just the test cluster.

My target customer is who didn't familiar with our replicated cli and want to pick his development job asap. The taskfile will give them all the shortcut for local development

Reply:
Great I get it, just update somewhere in the documentation so that it's clear that's what's happening. I can see the value where someone is using two different repositories waiting for testing to run and could get confused between which cluster they meant to act on. 👍